iree_tests/configs/config_sdxl_cpu_llvm_task.json

```diff
@@ -0,0 +1,17 @@
+{
+    "config_name": "sdxl_cpu_llvm_task",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=llvm-cpu",
+      "--iree-llvmcpu-target-triple=x86_64-linux-gnu",
+      "--iree-llvmcpu-target-cpu-features=host",
+      "--iree-llvmcpu-fail-on-out-of-bounds-stack-allocation=false",
+      "--iree-llvmcpu-distribution-size=32"
+    ],
+    "iree_run_module_flags": [
+      "--device=local-task"
+    ],
+    "skip_compile_tests": ["opt-125M", "resnet50", "sdxl-vae-decode-tank", "sdxl-prompt-encoder-tank"],
+    "skip_run_tests": ["opt-125M", "resnet50", "sdxl-vae-decode-tank", "sdxl-prompt-encoder-tank"],
+    "expected_compile_failures": [],
+    "expected_run_failures": []
+  }
```

Comment:
nit: A few of these files are missing a final newline character.

(If a file does not end in a newline character in Unix, it is not technically a file: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file. All tools I've seen are robust to that, but it's still the spec)

You can set your editor to automatically insert final newline characters (and also trim excess ones). For example, in VSCode: 
![image](https://github.com/nod-ai/SHARK-TestSuite/assets/4010439/29083e61-b35e-4297-89db-f48697411329)

Reply:
Cool, I've been manually checking and adding before merging a PR. This will help :). Thanks for the tip.